cypress/e2e/plugins/plugin-utils.js

```diff
@@ -66,13 +68,15 @@ function performPluginAction (action, plugin, pluginName) {
 function failAction (action) {
   cy.get('#plugin-action-button').click()
 
-  cy.get(panelCompleteQuery)
-    .should('not.be.visible')
-  cy.get(panelErrorQuery)
-    .should('not.be.visible')
-  cy.get(panelProcessingQuery)
-    .should('be.visible')
-    .contains(`${capitalize(action === 'upgrade' ? 'Upgrad' : action)}ing ...`)
+  if (Cypress.env('SKIP_PLUGIN_ACTION_INTERIM_STEP') !== 'true') {
```

Comment:
Yeah, I've had problems for a while.  On my laptop and my home internet connection the install happens too quickly and it fails because it's in the complete state and it should be in the pending state.